src/views/product-landing/components/heading/types.ts

```diff
@@ -0,0 +1,8 @@
+export interface HeadingProps {
+  slug: string
+  heading: string
+  /** Set the HTML heading level. */
+  level: 1 | 2 | 3 | 4 | 5 | 6
+  /** Optionally use a different heading-x style than the default for the semantic HTML heading level. Note that only 0-4 are supported, as these are the styles we have available from the design system. */
+  style_as?: 0 | 1 | 2 | 3 | 4
```

Comment:
Very interesting that this is zero-indexed 😄

Reply:
Bike shedding alert, feel free to ignore: would `size` be a good(er) name here? I think `styleAs` is just fine, but offering up `size` as I've seen it used in this way before.

Reply:
yeah +1 that the zero indexing is a little odd... i dig `size`, will update! 👍